src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -766,7 +766,9 @@ public void close() {
 								}
 								return keepGoing;
 							}
-							this.listener.onCommandAsync(receivedMessage, webSocket);
+							if (receivedMessage != null) {
```

Comment:
>receivedMessage [](start = 11, length = 15)

Under what circumstances does a null receiveMessage arrive? if it's null should keepGoing be true or false?

Reply:
Theoretically it should never happen, since the only possible cause is the control websocket closes. Just want to make sure it won't fall into the line below because it will cause exception.

---
In reply to: [261783674](https://github.com/Azure/azure-relay-java/pull/27#discussion_r261783674) [](ancestors = 261783674)

Reply:
I guess keepGoing should be true by default, unless closeCalled is true, which should be caught in the block above.

---
In reply to: [261791257](https://github.com/Azure/azure-relay-java/pull/27#discussion_r261791257) [](ancestors = 261791257,261783674)